src/jobs/dev-promote-prod-from-commit-subject.yml

```diff
@@ -119,9 +130,8 @@ steps:
         - run:
             name: construct/push git release tag
             command: |
-
               # construct/push new tag
-              NEW_VERSION=$(circleci orb info <<parameters.orb-name>> | grep Latest | sed -E 's|Latest: <<parameters.orb-name>>@||')
+              NEW_VERSION=$(echo ${ORB_VERSION}| sed -E 's|<<parameters.orb-name>>@||')
```

Comment:
This change is to get the version number saved from the output of the `circleci orb publish promote` command, to improve the accuracy of the orb version number that should be associated with this release